src/main/java/racingcar/RacingGame.java

```diff
@@ -8,43 +13,40 @@
 import static racingcar.view.ResultView.printCars;
 
 public class RacingGame {
+    private static final int INITIAL_TRY_NO = 0;
 
-    private final String carNames;
-    private final int tryNo;
-    private int gameCount;
+    private List<String> carNames;
+    private int tryNo;
 
-    public List<Car> carList = new ArrayList<>();
+    private List<Car> cars = new ArrayList<>();
```

Comment:
인스턴스 변수를 한번 할당한 뒤, 재할당 할 일이 없다면 아래와 같이 `final` 키워드를 사용해서 재할당을 방지해주면 좋을 것 같아요. 😃

```suggestion
    private final List<Car> cars;
    
    // 생성자 파라미터로 List<Car>를 주입받도록 수정
    public RacingGame(List<Car> cars, int tryNo) {
        this.cars = cars;
        this.tryNo = tryNo;
    }
```


RacingGame 클래스 내부의 `makeCar`는  RacingMain에서 아래와 같이 RacingGame의 생성자 파라미터로 CarFactory가 생성한 List<Car>를 주입받는 형태로 변경해보셨으면 어떨까 싶어요. 

```java
        String[] split = carNames.split(",");
        RacingGame racingGame = new RacingGame(CarFactory.createCar(split), tryNo);

        while (!racingGame.isEnd()) {
```